src/widgets/survey/model/hooks/useFlowRecordInitialization.ts

```diff
@@ -45,40 +51,48 @@ export function useFlowRecordInitialization({
     const isActivityFlow = entityType === 'flow';
 
     if (!isActivityFlow) {
+      const activity = applet.activities.find(x => x.id === entityId)!;
+
       return buildSingleActivityPipeline({
         appletId,
         eventId,
-        activityId: entityId,
+        activity: { id: activity.id, name: activity.name },
       });
     } else {
-      const activityIds = applet.activityFlows.find(
-        flow => flow.id === entityId,
-      )?.activityIds;
+      const activityIds = flow!.activityIds;
+
+      const activities = applet.activities
```

Comment:
Please add `IdName[]` type here instead of passing it to the map function

Reply:
Sorrym comment is not clear